deploy/terraform/outputs.tf

```diff
@@ -0,0 +1,6 @@
+
+output "web_public_dns" {
+  description = "The public DNS address of the web server"
+  value       = aws_lb.nia_gp2gp_elb.dns_name
+  depends_on  = [aws_lb.nia_gp2gp_elb]
```

Comment:
With terraform, we already have a reference to `aws_lb.nia_gp2gp_elb` on line 4 so terraform doesn't need this `depends_on` line.

See [depends_on docs](https://developer.hashicorp.com/terraform/language/meta-arguments/depends_on#processing-and-planning-consequences).